src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -173,6 +176,16 @@ export const QueryResultPane = () => {
 
     const gridRef = useRef<SlickGridHandle>(null);
 
+    const getExecutionPlanGraphs = async (contents: string) => {
+        if (!state?.state?.executionPlanGraphs!.length) {
```

Comment:
can you rename the context var to actually be `context`?  `state.state` is a bit misleading to read

Reply:
Also, if `context` is undefined, this entire component can just return immediately instead of getting down to here.

Reply:
Also also, is executionPlanGraphs ever undefined?  if so, that should be checked (and returned?) if it's undefined.  If not, maybe that property shouldn't be nullable.